src/main/java/org/springframework/integration/kafka/support/KafkaHeaders.java

```diff
@@ -28,8 +29,12 @@
 
 	public static final String MESSAGE_KEY = PREFIX + "messageKey";
 
-	public static final String PARTITION_ID = PREFIX + "_partitionId";
+	public static final String PARTITION_ID = PREFIX + "partitionId";
 
-	public static final String OFFSET = PREFIX + "_offset";
+	public static final String OFFSET = PREFIX + "offset";
```

Comment:
Are we going back to plan A (add this to 1.1.x only).

If not, unfortunately, we can't change these in 1.0.x (breaking change).